net.sourceforge.Ri-li.json

```diff
@@ -29,13 +29,17 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://downloads.sourceforge.net/project/ri-li/Ri-li%20Linux_Unix/Ri-li%20V2.0.1/Ri-li-2.0.1.tar.bz2",
+                    "url": "http://downloads.sourceforge.net/sourceforge/ri-li/Ri-li-2.0.1.tar.bz2",
```

Comment:
this should be available in https too. It's prefered.

Reply:
Oh yea, thanks.